lib/android.js

```diff
@@ -121,15 +121,15 @@ exports.detect = function detect(config, opts, finished) {
 
 			// scan various paths
 			var dirs = process.platform === 'win32'
-				? [ '%SystemDrive%', '%ProgramFiles%', '%ProgramFiles(x86)%', '%CommonProgramFiles%', '~' ]
-				: [ '/opt', '/opt/local', '/usr', '/usr/local', '~' ];
+				? [ '%SystemDrive%', '%ProgramFiles%', '%ProgramFiles(x86)%', '%CommonProgramFiles%', '~', '%LOCALAPPDATA%/Android' ]
```

Comment:
Can this and the other copy of paths be extracted to a common method/constant? We're defining the exact same paths to look at in both cases, aren't we?

It also seems like maybe the sdk/ndk search against these paths should just happen once. I assume there's no easy way to combine them, though.

Reply:
Pushed the change to move the dirs out to a common constant. I'm not too sure on the sdk/ndk search, (to me) reading the code it makes more sense to have them separated and I don't think there'd be too much of a gain from searching once?